plugins/rikki/heroeslounge/models/GameParticipation.php

```diff
@@ -57,4 +57,9 @@ public function getOrderedTalentsAttribute($value)
     {
         return $this->talents()->orderBy('talent_tier', 'asc')->get();
     }
+
+    public function isInSecondPickTeam()
+    {
+        return in_array($this->draft_order, [2, 3, 6, 7, 10]);
```

Comment:
If you are optimizing, this has a high overhead and could be replaced by draft_order == 2 || ... == 3 || ...